vendor/github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub/transport.go

```diff
@@ -228,7 +228,6 @@ func (t *Transport) startSubscriber(ctx context.Context, sub subscriptionWithTop
 	}
 	// Ok, ready to start pulling.
 	err := conn.Receive(ctx, func(ctx context.Context, m *pubsub.Message) {
-		logger.Info("got an event!")
```

Comment:
Instead of patching vendor, let's update the version of sdk-go to a more recent one that doesn't have this line.

Reply:
Looks like this was fixed in https://github.com/cloudevents/sdk-go/pull/189